added generator in metadata (#133)
dlr-eoc/ukis-pysat: CHANGELOG.rst

Changed
^^^^^^^

> - ``data``: generator added to query_metadata and query_metadata_srcid #127

Reviewer: ```suggestion
- ``data``: generator instead of catalog return for query_metadata and query_metadata_srcid #127
```
